Modules/_csv.c

```diff
@@ -882,13 +884,15 @@ Reader_iternext(ReaderObj *self)
 static void
 Reader_dealloc(ReaderObj *self)
 {
+    PyTypeObject *tp = Py_TYPE(self);
     PyObject_GC_UnTrack(self);
     Py_XDECREF(self->dialect);
     Py_XDECREF(self->input_iter);
     Py_XDECREF(self->fields);
     if (self->field != NULL)
         PyMem_Free(self->field);
     PyObject_GC_Del(self);
+    Py_DECREF(tp);
```

Comment:
`Py_Type()` can not increase the refcount, so we don't need to use `Py_DECREF(tp)` in here, no?